compiler-fuzz.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+make -C fuzz/compile-and-execute run
```

Comment:
this is a rust project, right? it would seem appropriate to use the `cargo` infrastructure for building and running fuzzers

Reply:
it's to embed commands, really.  i can make a build.rs if that satifies?

Reply:
I'm not following. Is there a makefile this is running? what does it do?
I would expect there to not be a makefile not this shell script, but that the fuzz targets are run by:
```
cargo fuzz run <name-of-target>
```

Reply:
some of the matierial i had read earlier suggested a special command line to run with asan etc enabled, so i had a special command run for that.  it's kind of neat that cargo fuzz is all inclusive like that.  thanks.

Reply:
the build system (`cargo`) handles all of that for you. you need the nightly build though, so in the other repos we run:
```
cargo +nightly fuzz run <name>
```